charts/loki-distributed/README.md.gotmpl

```diff
@@ -22,6 +22,9 @@ helm repo add grafana https://grafana.github.io/helm-charts
 
 Major version upgrades listed here indicate that there is an incompatible breaking change needing manual actions.
 
+### To 0.80.0
+Introduces a default `cluster_label` for the ring memberlist, which will temporarily disrupt ingestion as it rolls out, unless you temporarily set `cluster_label_verification_disabled`.
```

Comment:
could you we have a more descriptive note like: https://github.com/grafana/helm-charts/pull/3058/files#diff-692d159d251d994ada48f63a9a8d8d570ff515aec9a09c1d156544e786d3efa6R44-R45 ?